videos-list/10-Rob.md

```diff
@@ -0,0 +1,33 @@
+```txt
+
+Speaker: Rob Zinkov
+
+Title: A Tour of Model Checking techniques
+
+Event description:
+Have you ever written a model in PyMC3 and aren’t sure if it’s any good? In this talk I will show you the many ways you can evaluate how will your model fits your data using PyMC3. Not all these techniques may be applicable for your particular problem but you will definitely walk away with a few new tricks for being confident in the models you fit.
+
+Discourse Discussion
```

Comment:
@OriolAbril I think the Discourse link should be ok? The videos are now going on two years old.  And more people will be interested in watching if it's a topic they are searching for rather than engaging with the conference.

The goal of this added description to YouTube is to increase search engine optimization and represent the talk with more information so people find it when they are searching on the internet.  

People skim through descriptions quickly for information.   Don't worry that it's getting lost.  I know Discourse is important for the maintainers.  For users, they want to get to what they are looking for quickly which is the video on YouTube.  And the goal of the description is for search engine optimization.

Reply:
RE: PyMC
I was in touch with Thomas, and he prefers changing to PyMC for SEO.   And since we will be sharing the videos on social media, may as well, so as not to confuse people.

I suppose moving forward, consistency is important to reinforce the brand name.